packs/data/pathfinder-bestiary-3.db/bone-ship.json

```
@@ -620,7 +615,7 @@
                     "value": 2
                 },
                 "description": {
-                    "value": "<p>The crew let out an anguished cry in unison. This is a <em>@Compendium[pf2e.spells-srd.Wail of the Banshee]{Wail of the Banshee}</em> spell with @Template[type:emanation|distance:100] (DC 39). In addition, any creature within 5 feet of the ship is @Compendium[pf2e.conditionitems.Grabbed]{Grabbed} by the crew.</p>\n<p>The ship can't use Crew's Call again for [[/br 1d4+1 #rounds]]{1d4+1 rounds}.</p>"
+                    "value": "<p>The crew let out an anguished cry in unison. This is a <em>@Compendium[pf2e.spells-srd.Wail of the Banshee]{Wail of the Banshee}</em> spell with @Template[type:emanation|distance:100] (@Check[type:fortitude|dc:39|traits:magical,necromancy,auditory,death,negative|overrideTraits:true]). In addition, any creature within 5 feet of the ship is @Compendium[pf2e.conditionitems.Grabbed]{Grabbed} by the crew.</p>\n<p>The ship can't use Crew's Call again for [[/br 1d4+1 #Recharge Crew's Call]]{1d4+1 rounds}.</p>"
```

```suggestion
                    "value": "<p>The crew let out an anguished cry in unison. This is a <em>@Compendium[pf2e.spells-srd.Wail of the Banshee]{Wail of the Banshee}</em> spell with @Template[type:emanation|distance:100] (@Check[type:fortitude|dc:39|traits:magical,necromancy,auditory,death,negative]). In addition, any creature within 5 feet of the ship is @Compendium[pf2e.conditionitems.Grabbed]{Grabbed} by the crew.</p>\n<p>The ship can't use Crew's Call again for [[/br 1d4+1 #Recharge Crew's Call]]{1d4+1 rounds}.</p>"
```
Keep the divine trait on the spell